app/controllers/home_controller.rb

```diff
@@ -1,5 +1,14 @@
 class HomeController < ApplicationController
   def index
     @previous_item_data = Item.previous_item_data current_user
+    @all_lists = []
+    current_user.lists.each do |list|
+      @all_lists << list
+    end
+    current_user.shared_lists.each do |shared_list|
+      if list = List.find_by(id: shared_list.list_id)
+        @all_lists << list
+      end
+    end
```

Comment:
This looks like you should be able to refactor this down to `@all_lists = current_user.lists + current_user.user_shared_lists`.  We talked about this as a possibility, did something come up in testing where that wasn't possible?

Reply:
Sure can! I wrote this before our discussion and forgot about it.